functions.php

```diff
@@ -126,8 +126,31 @@ function thunderblog_customize_register ( $wp_customize ) {
 		)
 	);
 }
+
+/**
+ * Adds the featured image (if any) to the (rss/atom/whatever) feed.
+ *
+ * @param string $text
+ *
+ * @return string
+ */
+function thunderblog_add_featured_image_to_feed( string $text ): string {
+	global $post;
```

Comment:
This is just a "beautiful" WP-ism. When it iterates through the posts, it will set the global post variable to the current post.